pkg/statistics/handle/autoanalyze/priorityqueue/non_partitioned_table_analysis_job.go

```diff
@@ -212,6 +212,16 @@ func (j *NonPartitionedTableAnalysisJob) analyzeIndexes(
 	if len(j.Indexes) == 0 {
 		return true
 	}
+	autoAnalyzeVersion := sctx.GetSessionVars().AnalyzeVersion
+	if autoAnalyzeVersion == 1 {
```

Comment:
Please add some comments to explain why we need to Special treatment the auto analyze version 1 ?

Reply:
Updated. Thank you!